backend/tests/v5/unit/processors/teamspaces/projects/containers/containers.test.js

```diff
@@ -59,7 +59,7 @@ const containerSettings = {
 	},
 };
 
-const user1Favourites = [1];
+let user1Favourites = [1];
```

Comment:
use a different dataset if this needs to change, so you don't contaminate across tests